search-processors/src/test/java/org/opensearch/searchpipelines/questionanswering/generative/GenerativeQARequestProcessorTests.java

```diff
@@ -23,27 +23,30 @@
 
 import java.util.HashMap;
 import java.util.Map;
+import java.util.function.BooleanSupplier;
 
 public class GenerativeQARequestProcessorTests extends OpenSearchTestCase {
 
+    private BooleanSupplier alwaysOn = () -> true;
```

Comment:
Can we write the negative test cases also when the feature flag is not on then exception is thrown.

Reply:
Yes.